tests/phpunit/tests/Shortcode/Stories_Shortcode.php

```diff
@@ -145,10 +146,11 @@ public function test_prepare_story_attrs() {
 			'date'                      => 'true',
 			'excerpt'                   => 'true',
 			'archive_link'              => 'randomtext',
-			'archive_label'             => 'true',
+			'archive_label'             => 'Stories',
```

Comment:
```suggestion
			'archive_label'             => __( 'Stories', 'web-stories' ),
```

Reply:
This is a test file, no need for translation here.

Reply:
🤦